src/edit.js

```diff
@@ -42,6 +46,34 @@ export default function Edit( { attributes, setAttributes } ) {
 							} )
 						}
 					/>
+					<PanelColorSettings
```

Comment:
Thank you for working on this @nielslange 
Unfortunately, this is not how we want to use colors.
Your implementation only implements custom colors (as in, the color will always be a hex/rgb code).

In Gutenberg, you have the option between a relative color and a custom color. Relative colors are defined by a theme in a palette, and custom colors are custom colors.
Relative colors are applied by a className, custom colors are hardcoded directly.

I'd invite you to take a look into Gutenberg codebase, `packages/block-library/src` has all of Gutenberg blocks.
You can also check some of our atomic blocks like `js/atomic/block/product-elements/title`

You can also read this article I wrote a year ago about how to implement color controls in a block.
pca54o-6J-p2

Reply:
Thanks for your review and the suggestions, @senadir. I'll take a look at the resources you pointed out, to update this PR accordingly.

Reply:
@senadir: As requested, I've implemented the colors using `withColors` and `getColorClassName`. Implementing the font color and the background color was no problem. However, to implement the border color, I had to copy the following CSS definitions from Gutenberg into this PR:

```css
.has-pale-pink-border-color {
	border-color: #f78da7;
}
.has-vivid-red-border-color {
	border-color: #cf2e2e;
}
.has-luminous-vivid-orange-border-color {
	border-color: #ff6900;
}
.has-luminous-vivid-amber-border-color {
	border-color: #fcb900;
}
.has-light-green-cyan-border-color {
	border-color: #7bdcb5;
}
.has-vivid-green-cyan-border-color {
	border-color: #00d084;
}
.has-pale-cyan-blue-border-color {
	border-color: #8ed1fc;
}
.has-vivid-cyan-blue-border-color {
	border-color: #0693e3;
}
.has-vivid-purple-border-color {
	border-color: #9b51e0;
}
.has-white-border-color {
	border-color: #fff;
}
.has-very-light-gray-border-color {
	border-color: #eee;
}
.has-cyan-bluish-gray-border-color {
	border-color: #abb8c3;
}
.has-very-dark-gray-border-color {
	border-color: #313131;
}
.has-black-border-color {
	border-color: #000;
}
```

I saw that you created https://github.com/WordPress/gutenberg/issues/16848, but this hasn't been addressed yet. I wonder if my approach is the correct way or if you can think of a better way.